File: fix_og.js
const fs = require('fs');

const files = fs.readdirSync('.').filter(f => f.endsWith('.html'));

files.forEach(f => {
    let content = fs.readFileSync(f, 'utf8');
    const target = '<meta property="og:type" content="website">';
    const parts = content.split(target);
    if (parts.length > 2) {
        // If it appears more than once, remove the last occurrence
        const lastPart = parts.pop();
        const newContent = parts.join(target) + lastPart;
        fs.writeFileSync(f, newContent);
        console.log("Fixed " + f);
    }
});

File: fix_og.py
import os

for f in os.listdir('.'):
    if f.endswith('.html') and os.path.isfile(f):
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
            
        count = content.count('<meta property="og:type" content="website">')
        if count > 1:
            parts = content.rsplit('<meta property="og:type" content="website">', 1)
            # Remove any trailing whitespace left over by the removed tag
            new_content = parts[0] + parts[1]
            with open(f, 'w', encoding='utf-8') as file:
                file.write(new_content)
            print(f"Fixed {f}")
